tfe/data_source_oauth_client_test.go

```diff
@@ -0,0 +1,52 @@
+package tfe
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccTFEOAuthClientDataSource_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTFEOAuthClientDataSourceConfig(),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttrPair(
+						"tfe_oauth_client.test", "api_url",
+						"data.tfe_oauth_client.client", "api_url"),
+					resource.TestCheckResourceAttrPair(
+						"tfe_oauth_client.test", "http_url",
+						"data.tfe_oauth_client.client", "http_url"),
+					resource.TestCheckResourceAttrPair(
+						"tfe_oauth_client.test", "oauth_token_id",
+						"data.tfe_oauth_client.client", "token_id"),
```

Comment:
just matching naming suggestion:
```suggestion
						"data.tfe_oauth_client.client", "oauth_token_id"),
```